Duplicati/Library/Encryption/GPGEncryption.cs

```diff
@@ -95,8 +98,8 @@ public class GPGEncryption : EncryptionBase
         /// <summary>
         /// The PGP program to use, should be with absolute path
         /// </summary>
-        private readonly string m_programpath = "gpg";
-
+        private string m_programpath { get; set; } = GetGpgProgramPath();

```

Comment:
Is this called often (e.g., for each encrypt/decrypt operation), and is it potentially expensive?  Would it make sense to use a Lazy initializer for this?

Reply:
The check is fast and we would want to handle the case when a user installs GPG while Duplicati is running.